src/main/java/dev/j3fftw/litexpansion/ItemSetup.java

```diff
@@ -128,6 +145,18 @@ private void registerMiscItems() {
             new ItemStack(Material.REDSTONE), new ItemStack(Material.LAPIS_LAZULI), new ItemStack(Material.REDSTONE)
         );
 
+        registerItem(Items.ENERGY_CRYSTAl, RecipeType.ENHANCED_CRAFTING_TABLE,
+            new ItemStack(Material.REDSTONE), new ItemStack(Material.REDSTONE), new ItemStack(Material.REDSTONE),
+            new ItemStack(Material.REDSTONE), new ItemStack(Material.DIAMOND), new ItemStack(Material.REDSTONE),
+            new ItemStack(Material.REDSTONE), new ItemStack(Material.REDSTONE), new ItemStack(Material.REDSTONE)
+        );
```

Comment:
There is a Power Crystal in Slimefun already which is based off of IC2's one.

Reply:
oh i didnt know that